src/listeners/QuoteSystem.hs

```diff
@@ -36,6 +47,18 @@ removeQuote name = do
     newTable <- HM.delete name <$> readHashMapDB quotesTable
     writeHashMapDB quotesTable newTable
 
+randomQuote :: IO (T.Text, IO (Maybe T.Text))
+randomQuote = do
+    keys <- HM.keys <$> readHashMapDB quotesTable
+    let l = length keys
+    g <- getStdGen
```

Comment:
`g` is not being used? The global IO generator is being used so there's no need to get a generator here! :)